tests/test_logger.py

```diff
@@ -1,9 +1,12 @@
+from unittest.mock import patch
```

Comment:
Hi, @pared, thank you so much for your PR!

We are supporting Python 2.7, so it's better to use the `mock` module instead of `unittest.mock`:

```suggestion
from mock import patch
```

Reply:
right, sorry about that